lib/iris/tests/idiff.py

```diff
@@ -220,7 +220,7 @@ def step_over_diffs(result_dir, action, display=True):
         count = len(results)
 
         for count_index, result_fname in enumerate(results):
-            key = os.path.splitext("-".join(result_fname.split("-")[1:]))[0]
+            key = os.path.splitext("-".join(result_fname.split("result-")[1:]))[0]
```

Comment:
I agree this deserves a small "What's New" entry in the `Internal` section.

Reply:
done, see what you think